src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -185,10 +177,10 @@ private Predicate<Person> preparePredicate(String userInput) {
     }
 
     private boolean isNumeric(String str) {
-        return str.matches("\\d+");
+        return str.matches("\\d+") && str.length() != 6;
```

Comment:
This code appears to be repeated. Perhaps you could import this method and use it following the DRY principle?